src/controller/model/endpoint.ts

```diff
@@ -897,6 +897,33 @@ class Endpoint extends Entity {
             }
         }
     }
+
+    public async generalCommand(
```

Comment:
Can we call this `zclCommand` and refactor the other zcl calls to use this method? E.g. `readResponse` and `writeStructured` can be easily refactored to this, probably also the others.